Src/ILGPU/Runtime/ExchangeBuffer.cs

```diff
@@ -177,17 +179,33 @@ protected override void Dispose(bool disposing)
         /// </summary>
         public TIndex Extent { get; protected set; }
 
+        /// <summary>
+        /// Internal array view
+        /// </summary>
+        [SuppressMessage(
+            "Microsoft.Design",
+            "CA1051: Do not declare visible instance fields",
+            Target = "target")]
+        protected ArrayView<T, TIndex> cpuView;
```

Comment:
MINOR: Move this from `Properties` region to `Instance` region.

Actually, maybe let's keep it here, and instead of ignoring the warning, let's change this field into a property.

```suggestion
        protected ArrayView<T, TIndex> CpuArrayView { get; set; }
```

Reply:
Yeah I don't know why I didn't do this to begin with. So much cleaner.

Reply:
However, I feel the name should remain CPUView to stay consistent with the View property and (I believe) what it was before. What are your thoughts?

Reply:
I was thinking to call the protected property `CPUArrayView`, and the public property `CPUView`. That way we can add the `[Obsolete]`attribute to the public `CPUView` property. And in future, if we remove `CPUView`, we can leave the protected property unchanged.

Reply:
Actually yeah, that sounds good. I didn't really think about breaking changes here, so that's probably something I should've considered.